primitives/evm/src/account_provider.rs

```diff
@@ -22,14 +22,14 @@ pub trait AccountProvider {
 	/// for particular account.
 	type Index: AtLeast32Bit;
 
-	/// Creates a new account in accounts records.
+	/// Creates a new contract account record in accounts records.
```

Comment:
Repetition (record record), better rephrase it so it's less confusing

Reply:
In general decided to remove `_record` postfix